sopel_unicode/plugin.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import annotations
```

Comment:
Probably want a basic file header here? Project name, one-line description, copyright, license

As an example I'll link one of my recently updated personal plugins, [`sopel-dns`](https://github.com/dgw/sopel-dns/blob/b185fb789344c59498e4b4bcd4be77ac2fbd80ca/sopel_dns/plugin.py#L1-L9).